core/src/test/java/org/infinispan/test/Exceptions.java

```diff
@@ -30,7 +30,8 @@ public static void assertException(Class<? extends Throwable> exceptionClass, St
       assertException(exceptionClass, t);
       Pattern pattern = Pattern.compile(messageRegex);
       if (!pattern.matcher(t.getMessage()).matches()) {
-         fail("Wrong exception message: " + t.getMessage());
+         t.printStackTrace();
```

Comment:
How about throwing something here? This way Surefire will print out the stacktrace.